elasticsearch/src/main/scala/com/stratio/crossdata/connector/elasticsearch/ElasticSearchQueryProcessor.scala

```diff
@@ -126,19 +140,19 @@ class ElasticSearchQueryProcessor(val logicalPlan: LogicalPlan, val parameters:
 
   private[this] def checkNativeFilters(filters: Array[SourceFilter]): Boolean = filters.forall {
     case _: sources.EqualTo => true
-        case _: sources.In => false
-        case _: sources.LessThan => false
-        case _: sources.GreaterThan => false
-        case _: sources.LessThanOrEqual => false
-        case _: sources.GreaterThanOrEqual => false
-        case _: sources.IsNull => false
-        case _: sources.IsNotNull => false
-        case _: sources.StringStartsWith => false
-        case _: sources.StringEndsWith => false
-        case _: sources.StringContains => false
-        case sources.And(left, right) => checkNativeFilters(Array(left, right))
-        case sources.Or(left, right) => checkNativeFilters(Array(left, right))
-        // TODO add more filters (Not?)
+    case _: sources.In => true
+    case _: sources.LessThan => true
+    case _: sources.GreaterThan => true
+    case _: sources.LessThanOrEqual => true
+    case _: sources.GreaterThanOrEqual => true
+    case _: sources.IsNull => true
+    case _: sources.IsNotNull => true
+    case _: sources.StringStartsWith => true
+    case _: sources.StringEndsWith => false
+    case _: sources.StringContains => true
+    case sources.And(left, right) => checkNativeFilters(Array(left, right))
+    case sources.Or(left, right) => false //checkNativeFilters(Array(left, right))
```

Comment:
Unnecessary since the default match (`case _`) covers this case with the same result.